attack.py

```diff
@@ -139,20 +137,14 @@ def mainth(self):
                         response = scraper.get(site, timeout=10)
                         self.write_statistic_success(site, response.status_code)
                         del response
-            except ConnectionError:
-                self.write_statistic_error(site)
-                continue

```

Comment:
except ConnectionError and except Exception do the same thing. There is no need to catch ConnectionError